cpp.md

```diff
@@ -523,3 +523,44 @@ sort(v.begin(), v.end(), greater<int>()); // sorting in the vector in descending
 |#include<graphics.h> | It is used include and facilitate graphical operations in program|
 |#include<bits/stdc++.h> | It is used to include all the standard library files|
 |#include <bits/stdc++.h> | Inlcude all the headers. A Usefull Hack for Competitive Programming|
+### Array of structures
+Array of structure, is mixture of two cpp concepts,where we create a array of type structure
+here,each element of the array is basically a structure,which can store different data values depending on,how it is defined
+Example:
+```c
```

Comment:
code not running fine https://onecompiler.com/cpp/3yj87yjbt
please fix the errors